runtime/src/accounts_db.rs

```diff
@@ -7298,8 +7302,22 @@ impl AccountsDb {
 
                 for (slot, sub_storages) in snapshot_storages.iter_range(&range_this_chunk) {
                     scanner.set_slot(slot);
-                    if let Some(sub_storages) = sub_storages {
+
+                    let mut ancient = false;
+                    let (_, scan) = measure!(if let Some(sub_storages) = sub_storages {
+                        if let Some(storage) = sub_storages.first() {
```

Comment:
seems we can probably move this block to after the `scan_multiple_account_storages_one_slot` call, and just get ancient out of the measure block that way; let's us remove the mut.